gulp/docs.js

```diff
@@ -9,12 +10,112 @@ var path = require('path');
 var tar = require('gulp-tar');
 var webpack = require('webpack');
 var webpackConfig = require('../webpack.config.js');
+var CONFIG = require('./config');
+
+const isReactComponent = value =>
+	typeof value === 'function' &&
```

Comment:
are the indents weird here?

Reply:
i agree, but this is what prettier wants :P